client/packages/inventory/src/Stocktake/DetailView/AppBarButtons.tsx

```diff
@@ -31,14 +33,44 @@ export const AppBarButtonsComponent: FC<AppBarButtonProps> = ({
   const { data } = useStocktake.document.get();
   const { print, isPrinting } = useReport.utils.print();
 
+  const [isReportPrinting, setIsReportPrinting] = useState(false);
```

Comment:
The changes in here aren't required - there's a `useCallback` in the report selector which doesn't have any dependencies.. this is causing your problem!

Try this suggestion: [2276-fix-stocktake-report-sorting-suggestion](https://github.com/openmsupply/open-msupply/compare/2276-fix-stocktake-report-sorting-issue...2276-fix-stocktake-report-sorting-suggestion) - it's working for me.

Reply:
Ah... I see.. Nice. Updated bec90a205 and its working for me too. Thanks for the idea.